shogun-boot/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+if [ x"${JAVA_ENABLE_DEBUG}" != x ] && [ "${JAVA_ENABLE_DEBUG}" != "false" ]; then
+  JAVA_DEBUG_ARGS="-Xdebug -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=${JAVA_DEBUG_PORT:-5005}"
+fi
+
+java $JAVA_DEBUG_ARGS -Djava.security.egd=file:/dev/./urandom -jar /opt/app.jar --spring.config.location=/config/application.yml
```

Comment:
May it makes sense to include extra `$JAVA_OPTS` here, or should `openjdk:11-jdk-slim``s `JAVA_TOOL_OPTIONS` property be used for e.g. increase of java heap space

Reply:
Yes, the `JAVA_TOOL_OPTIONS` should be used here.

Reply:
An example configuration is introduced [here](https://github.com/terrestris/shogun-docker/pull/4).